SPECS/mlnx-ofa_kernel-hwe/mlnx-ofa_kernel-hwe.spec

```
@@ -85,9 +85,10 @@
 
 %global base_name mlnx-ofa_kernel
 %{!?_name: %global _name %{base_name}-hwe}
-%{!?_version: %global _version 24.10}
-%{!?_release: %global _release OFED.24.10.0.7.0.1}
+%{!?_version: %global _version 25.07}
+%{!?_release: %global _release OFED.25.07.0.9.7.1}
 %global _kmp_rel %{_release}%{?_kmp_build_num}%{?_dist}
+%global MLNX_OFA_DRV_SRC 24.10-0.7.0
```

This looks like the old version. Shouldn't it be this:

```suggestion
%global MLNX_OFA_DRV_SRC 25.07-0.9.7.1
```

Same question for the HWE version of this spec.